landlab/components/bedrock_landslider/bedrock_landslider.py

```diff
@@ -0,0 +1,692 @@
+                        self.landslides_volume.append(storeV)
+                    if self.landslides_volume_sed is not None:
+                        self.landslides_volume_sed.append(storeV_sed)
+                    if self.landslides_volume_bed is not None:
+                        self.landslides_volume_bed.append(storeV_bed)
+
+            if i_slide.size > 0:
+                i_slide = np.delete(i_slide, 0, 0)
+
+        return suspended_Sed
+
+    def _landslide_runout(self, dt):
+        """
+        calculate landslide runout using a non-local deposition algorithm based on:
+            * Carretier S., Martinod P., Reich M., Godderis Y. (2016) Modelling
+              sediment clasts transport during landscape evolution.
+              Earth Surf Dyn: 4(1):237–51.
+            * Campforts B., Shobe C.M., Steer P., Vanmaercke M., Lague D., Braun J.
+              (2020) HyLands 1.0: a hybrid landscape evolution model to simulate
+              the impact of landslides and landslide-derived sediment on landscape
+              evolution. Geosci Model Dev: 13(9):3863–86.
+
+        Parameters
+        ----------
+        dt : float
+            timestep.
+
+        Returns
+        -------
+        dH_Hill : float
+            Hillslope erosion over the simulated domain.
+        V_leaving : float
+            Total volume of sediment leaving the simulated domain.
+        Qs_coreNodes : float
+            Sediment flux over the simulated domain.
+
+        """
+        z = self.grid.at_node["topographic__elevation"]
+        br = self.grid.at_node["bedrock__elevation"]
+        H = self.grid.at_node["soil__depth"]
+        sed_flux = self.grid.at_node["LS_sediment__flux"]
+        stack_rev = np.flip(self.grid.at_node["flow__upstream_node_order"])
+        ls_depo = self.grid.at_node["landslide__deposition"]
+        ls_sed_in = self.grid.at_node["landslide_sediment_point_source"]
+        node_status = self.grid.status_at_node
+
+        # Only process core nodes
+        stack_rev_sel = stack_rev[node_status[stack_rev] == NodeStatus.CORE]
+        receivers = self.grid.at_node["hill_flow__receiver_node"]
+        fract = self.grid.at_node["hill_flow__receiver_proportions"]
+
+        # keep only steepest slope
+        slope = np.max(self.grid.at_node["hill_topographic__steepest_slope"], axis=1)
+        slope[slope < 0] = 0
+
+        Qs_in = ls_sed_in * dt  # Qs_in, in m3 per timestep
+
+        # L following carretier 2016
+        L_Hill = np.matlib.divide(
+            self.grid.dx,
+            (
+                1
+                - np.matlib.minimum(
+                    np.matlib.square(np.matlib.divide(slope, self._angle_int_frict)),
+                    0.999,
+                )
+            ),
+        )
+        Qs_out = np.zeros(z.shape)
+        dH_Hill = np.zeros(z.shape)
+        H_i_temp = np.array(z)
+        max_D = np.zeros(z.shape)
+        max_dH = np.ones(z.shape) + np.inf
```

Comment:
If this is *inf*, can we just remove it?

Reply:
Good point, I had it there for some functionality I might support in the future, but not just yet. Removed it.